config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
 
-  root "users#index"
-
   resources :users do
   end
+
+  # Defines the root path route ("/") to the public_recipes controller's index action
+  root 'public_recipes#index'
```

Comment:
- [ ] You have made the `public recipe` page as the root page, but according to the project requirement, you have links on `navigation menu` that leads to all other links including `public recipe` page.
   -  For now, you could add a button _before or after_ Welcome message in `users index` page, and that would link to your `public recipe` page, and later place the link on navigation menu when you add navigation menu [in next milestone]
   - routes users index as the root page [ root "users#index" ]